services/predictors/layoutparser/mypy.ini

```diff
@@ -0,0 +1,11 @@
+[mypy]
+check_untyped_defs = True
+no_implicit_optional = True
+plugins = pydantic.mypy
+exclude = mmda/
```

Comment:
Ignore errors from the base `mmda` code, since I don't want to enforce them at the `services` level. There ought to be checks at the top level, after which I can remove this.